lib/cmd/common/local.js

```diff
@@ -53,6 +53,7 @@ var http = require("http"),
   async = require('async'),
```

Comment:
That's another case of `var x=1, y=2, z=3...;` that we need to decide on the style guide how we should go about. I personally prefer one `var` keyword per variable, too.